index.markdown

```diff
@@ -333,11 +333,11 @@ redirect_from: '/moveit/'
                         <span class="label label--orange">Released</span>
                     </div>
                     <div class="version-date">
-                        <p><span>Version:</span> 2.0 Alpha</p>
+                        <p><span>Version:</span> 2.0</p>
```

Comment:
Why should we remove the Alpha/Beta labels? This would look like we had published a major release already.

Reply:
Because it says it right above:
![image](https://user-images.githubusercontent.com/561060/69766648-1e270000-1136-11ea-8ee0-9305e7725f06.png)